README.md

```diff
@@ -128,7 +128,7 @@ Have a bug or a feature request? Please first read the [issue guidelines](https:
 
 Bootstrap's documentation, included in this repo in the root directory, is built with [Hugo](https://gohugo.io/) and publicly hosted on GitHub Pages at <https://v5.getbootstrap.com/>. The docs may also be run locally.
 
-Documentation search is powered by [Algolia's DocSearch](https://community.algolia.com/docsearch/). Working on our search? Be sure to set `debug: true` in `site/assets/js/src/search.js` file.
+The documentation search is powered by [Algolia's DocSearch](https://community.algolia.com/docsearch/). Working on our search? Be sure to set `debug: true` in the `site/assets/js/src/search.js` file.
```

Comment:
the addition of the second "the" here is probably the only one i'd say yes, it's a good change. all others i'd say are not essential/matter of opinion/not correct

Reply:
Feel free to dismiss it. I don't want to mark this as invalid since it doesn't seem like the other PRs we got so far :)